components/layout_2020/flow/line.rs

```diff
@@ -32,7 +32,7 @@ use crate::style_ext::PaddingBorderMargin;
 use crate::ContainingBlock;
 
 pub(super) struct LineMetrics {
-    /// The block offset of the line start in the containing [`InlineFormattingContext`].
+    /// The block offset of the line start in the containing Inline Formatting Context.
```

Comment:
I think you can just qualify the reference to InlineFormattingContext instead of removing the link.